sycl/include/sycl/detail/pi_error.def

```diff
@@ -97,6 +97,8 @@ _PI_ERRC(PI_ERROR_INVALID_VA_API_MEDIA_ADAPTER_INTEL, -1098)
 _PI_ERRC(PI_ERROR_INVALID_VA_API_MEDIA_SURFACE_INTEL, -1099)
 _PI_ERRC(PI_ERROR_VA_API_MEDIA_SURFACE_ALREADY_ACQUIRED_INTEL, -1100)
 _PI_ERRC(PI_ERROR_VA_API_MEDIA_SURFACE_NOT_ACQUIRED_INTEL, -1101)
+// backend is lost, e.g. it was already unloaded
```

Comment:
nit: UNITIALIZED seems suitable for other scenarios as well, like backend is not found, or a library is not system.